int32 y_min
int32 x_min
int32 y_max
int32 x_max
int32 categ
float32 score
float32 belief
string text